.github/workflows/default.yml

```diff
@@ -136,42 +119,46 @@ jobs:
       - name: Test
         run: make test-ui
 
+      - name: Build UI
+        run: make ui
+
+      - name: Cache dist
+        uses: actions/cache/save@v3
+        id: cache-dist
+        with:
+          path: dist
+          key: ${{ runner.os }}-${{ github.sha }}-dist
+
       - name: Porcelain
         run: |
           test -z "$(git status --porcelain)" || (git status; git diff; false)
 
   integration:
     name: Integration
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'push' || github.event_name == 'pull_request' }}
 
     steps:
       - uses: actions/checkout@v3
 
+      - uses: erezrokah/setup-go@feat/add_cache_prefix
+        with:
+          go-version: ^1.20
+          cache-key-prefix: integration-cache-
```

Comment:
Hier würde ich sogar sagen, dass der Go Build hier der gleicht ist wie der "normale" Build- wir könnten also auch den gleichen Cache Key nutzen. In beiden Fällen kommt ein Binary raus?

Reply:
Du hattest für die Tests auch ein eigenes Präfix genommen. Daher hatte ich das hier auch gemacht. Wenn da nichts technisch gegenspricht gerne gleich machen.

Reply:
Yep. Bei Test brauche es das weil da nur ein halber Cache raus kommt der denn immer wieder restored wird und mehr Zeit braucht. Hier ists ein voller Build wie im Buildschritt auch, daher könnte es der gleiche Cache sein.